extra_data/cli/make_virtual_cxi.py

```diff
@@ -100,7 +105,7 @@ def main(argv=None):
     inc_suspect = not args.exc_suspect_trains
     run = RunDirectory(run_dir, inc_suspect_trains=inc_suspect)
 
-    _, det_class = identify_multimod_detectors(run, single=True)
+    _, det_class = identify_multimod_detectors(run, detector_name=args.detector_name, single=True)
```

Comment:
Hi James,
I'm afraid this would not work in the current implementation, because this argument, 'detector_name', is not currently used by the 'identify_multimod_detectors' function at all :sweat: